templating.rst

```diff
@@ -4,21 +4,19 @@
 Creating and Using Templates
 ============================
 
-As you know, the :doc:`controller </controller>` is responsible for
-handling each request that comes into a Symfony application. In reality,
-the controller delegates most of the heavy work to other places so that
-code can be tested and reused. When a controller needs to generate HTML,
+As explained in :doc:`the previous article </controller>`, controllers are
```

Comment:
You are right ... except that in this case this article is indeed a part of a series. The Getting Started section groups the basic articles to learn Symfony and they are all linked. You can see that at the bottom of each article, which encourage s you to keep reading the next articles.